lib/shared/addon/components/cluster-driver/driver-rke/component.js

```diff
@@ -230,6 +244,43 @@ export default InputTextFile.extend(ClusterDriver, {
     }
   }),
 
+  windowsSupportChange: observer('windowsSupport', function() {
+    const windowsSupport = get(this, 'windowsSupport')
+
+    if (windowsSupport) {
+      let res = NETWORKCHOICES.filter((n) => n.value === 'flannel')
+
+      setProperties(this, {
+        networkContent:                                res,
+        'config.network.plugin':                       'flannel',
+        'config.network.options.flannel_backend_type': 'host-gw',
+      })
+    } else {
+      setProperties(this, {
+        networkContent:                                NETWORKCHOICES,
+        'config.network.options.flannel_backend_type': null,
```

Comment:
set `config.network.plugin` back to default choice (`flannel`), also add the default choice as a constant in the top of the file. we reference it a few times so we should make it a const.